src/modules/products/ui/components/product-card.tsx

```
@@ -0,0 +1,105 @@
+            src={imageUrl || "/placeholder.png"}
+            alt={name}
+            fill
+            className="object-cover"
+          />
+        </div>
+        <div className="p-4 border-y flex flex-col flex-1 gap-3">
+          <h2 className="text-lg font-medium line-clamp-4">{name}</h2>
+          <div
+            className="flex items-center gap-2"
+            onClick={(e) => e.stopPropagation()}
+          >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)